spec/features/merchants/items/index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "items index page", type: :feature do
+  before(:each) do
+    @merchant_1 = Merchant.create!(name: "Schroeder-Jerde", status: nil)
+    @merchant_2 = Merchant.create!(name: "John Johnson", status: nil)
+    @item_1 = @merchant_1.items.create!(name: "Ball", description: "round", unit_price: 75106, status: nil)
```

Comment:
I did these a bit differently however this works fine.  I'll likely accept your changes in the merge conflict for continuity.